File: ClientApp/src/app/home/post.ts
export interface IPost {
  title: string;
  views: number;
  content: string;
  excerpt: string;
  coverImagePath: string;
  isPublic: boolean;
}

export class Post implements IPost {
  constructor(public title: string,
    public views: number,
    public content: string,
    public excerpt: string,
    public coverImagePath: string,
    public isPublic: boolean) {}
}

File: Models/Post.cs
using Newtonsoft.Json;

namespace Blog.Models
{
    public class Post : Model{
        [JsonProperty("title")]
        public string Title{get; set;}
        [JsonProperty("views")]
        public int Views {get;set;} = 0;
        [JsonProperty("content")]
        public string Content {get; set;}
        [JsonProperty("excerpt")]
        public string Excerpt {get; set;}
        [JsonProperty("coverImagePath")]
        public string CoverImagePath {get; set;}
        [JsonProperty("isPublic")]
        public bool Public {get; set;}

    }
}